packages/ketcher-core/src/infrastructure/services/helpers.ts

```diff
@@ -22,7 +22,7 @@ export function getLabelRenderModeForIndigo() {
 
   return (
     renderModeMapping[
-      ketcherProvider.getKetcher().editor.options().showHydrogenLabels
-    ] || IndigoShowHydrogenLabelsMode.OFF
+      ketcherProvider.getKetcher()?.editor.options().showHydrogenLabels
+    ] || IndigoShowHydrogenLabelsMode.TERMINAL_HETERO
```

Comment:
can we fix use the new default? I think it makes sense we need `getKetcher()?`

Reply:
oh actually ignore that. I think the hydrogen atoms is something we did manually change